static/js/asideShowTimes.js

```diff
@@ -87,7 +86,14 @@ function createHTML(days, dates) {
             const a = document.createElement("a");
 
             a.href = "./movie.html#" + days[i][j].index;
```

Comment:
The link here should be changed to `/movie/id`

Reply:
Will fix!